migrations/0002_create_micro_deposit_transfers.up.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE `micro_deposit_transfers` (
+  `micro_deposit_id` varchar(40) NOT NULL,
+  `transfer_id` varchar(40) NOT NULL,
+  PRIMARY KEY (`transfer_id`)
```

Comment:
nit:
Define `transfer_id varchar(40) PRIMARY KEY` first; and I don't think we need to specify `not null` on PKs

Reply:
It's mostly the schema dump of current DB.

https://stackoverflow.com/a/3503124 - here is a discussion on this topic why or why no :)